styles/stylesheet.css

```diff
@@ -0,0 +1,254 @@
+}
+/********** HEADER **********/
+header {
+  background: black url(../images/header-image.jpg) center no-repeat; /* shorthand for: bg-color bg-image position/bg-size bg-repeat bg-origin bg-clip bg-attachment initial|inherit  */
+  height: 20rem;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  background-size: cover;
+  color: white;
+}
+header h1 {
+  margin: 0;
+}
+header h1 img {
+  height: 6rem;
+  margin-left: 3rem;
+  margin-top: 2rem;
+}
+header p {
+  margin-left: 3rem;
+  margin-top: 0;
+  font-style: italic;
+  color: lightgreen;
+  font-size: 1rem;
+}
+header img {
+  height: 2rem;
+}
+.checkout {
+  margin-top: 2rem;
+}
+.checkout div {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: baseline;
+}
+.cost {
+  margin-right: 3rem;
+  margin-left: 0;
+  text-align: right;
+  color: white;
+  font-style: normal;
+}
+button, .checkout-page input[type=submit] {
+  background-color: lightgreen;
+  color: white;
+  font-size: 1rem;
+  width: 8rem;
+  margin-right: 3rem;
+  border: none;
+  border-radius: 2px;
+  cursor: pointer;
+  transition-duration: 0.4s;
+  text-align: center;
+}
+button:hover, .checkout-page input[type=submit]:hover {
+  background-color: green;
+}
+.hidden {
+  position: absolute;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  height: 1px; width: 1px;
+  margin: -1px; padding: 0; border: 0;
+}
+/********** MENU NAV **********/
+.menu ul {
+  list-style-type: none; /* removes bullet points */
+  margin: 0; /* removes browser defaults */
+  padding: 0 100px;
+  overflow: hidden;
+  background-color: lightgrey;
+  font-weight: bold; /* makes text bold */
+  width: auto;
+}
+.menu li {
+  display: inline-block;
+  padding: 12px 16px 12px 0;
+}
+.menu li a {
+  color: black;
+}
+.menu li a:hover {
+  color: lightgreen;
+}
+/********** PAGE ARTICLE **********/
+.main-content {
+  display: grid;
+  margin-left: 6rem;
+  margin-top: 2rem;
+  margin-right: 6rem;
+  margin-bottom: 2rem;
+}
+.summary {
+  grid-area: 1 / 1 / 2 / 4;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-bottom: 2rem;
+}
+.summary div {
+  margin-left: 2rem;
+}
+.summary img {
+  height: 15rem;
+}
+/********** BLOG PAGE **********/
+.blog {
+  margin-left: 6rem;
+  margin-top: 2rem;
+}
+.sec-1 {
+  grid-area: 1 / 1 / 2 / 4;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-bottom: 2rem;
+}
+.sec-2 {
+  grid-area: 2 / 1 / 3 / 4;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-bottom: 2rem;
+}
+.sec-3 {
+  grid-area: 3 / 1 / 4 / 4;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-bottom: 2rem;
+}
+.sec-1 div, .sec-2 div, .sec-3 div {
+  margin-left: 2rem;
+}
+.sec-1 img, .sec-2 img, .sec-3 img {
+  height: 15rem;
+}
+/********** DOG LIST **********/
+.page-content {
+  grid-area: 2 / 1 / 3 / 4;
+  grid-gap: 3rem;
+  display: flex;
+  flex-flow: row wrap;
```

Comment:
Your grid looks awesome. Well done getting the columns to scale on different screen sizes